backend/Application/Api/GraphQL/Contracts/Contract.cs

```diff
@@ -0,0 +1,19 @@
+using Application.Api.GraphQL.Accounts;
+using HotChocolate.Types.Relay;
+
+namespace Application.Api.GraphQL.Contracts
+{
+    public class Contract
+    {
+        [ID]
+        public long Id { get; set; }
+        public ContractAddress ContractAddress { get; set; }
+        public string ModuleRef { get; set; }
+        public long Balance { get; set; }
+        public long FirstTransactionId { get; set; }
```

Comment:
is this going to be the transaction that created it? If not, please add that transaction as well, since it is a useful one.

Reply:
yes. this is the init transaction id